src/libraries/System.Private.CoreLib/src/System/IO/Strategies/WindowsFileStreamStrategy.cs

```diff
@@ -11,7 +11,6 @@ namespace System.IO.Strategies
     internal abstract class WindowsFileStreamStrategy : FileStreamStrategy
     {
         protected readonly SafeFileHandle _fileHandle; // only ever null if ctor throws
-        protected readonly string? _path; // The path to the opened file.
```

Comment:
we need a set of very similar changes for the `Net5CompatFileStreamStrategy` strategy as well:

https://github.com/dotnet/runtime/blob/3ff04916e222865c542b0c44ad3bbe783813c99e/src/libraries/System.Private.CoreLib/src/System/IO/Strategies/Net5CompatFileStreamStrategy.cs#L22-L23